### DISCUSSION TITLE
How do you auto-start the tour?

### DISCUSSION THREAD

**Discussion by ghost** [gilbarbara/react-joyride#719] [Q&A] [answered]
Hello, I would like to have the tour started automatically without the user having to click the red pulsating button.

I have dug through the whole documentation, but haven't find such option.

Is such option available in the current version?

Anyways, this is amazing plugin. ☝️☝️

**Comment by venumanthena** [comment, marked as answer]
In your steps array, add a key as disableBeacon: true  to get rid of the red pulsating button :)

**Reply by younesrivandi** [reply]
thank you
